src/Renci.SshNet/Common/ChannelInputStream.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using System.IO;
+
+using Renci.SshNet.Channels;
+
+namespace Renci.SshNet.Common
+{
+    /// <summary>
+    /// ChannelInputStream is a one direction stream intended for channel data.
+    /// </summary>
+    public class ChannelInputStream : Stream
```

Comment:
Since the constructor is internal, perhaps the type could just be internal unless there is a reason to make it public

Reply:
Sure.